src/CMakeLists.txt

```diff
@@ -360,21 +369,23 @@ add_subdirectory(test)
 #
 ## Setup benchmark_hotgym
 #
-
-source_group("examples" FILES examples/Hotgym.cpp examples/HelloSPTP.cpp)
+source_group("examples" FILES 
+    examples/hotgym/Hotgym.cpp 
+    examples/hotgym/HelloSPTP.hpp
+    )
 
 set(src_executable_hotgym benchmark_hotgym)
-add_executable(${src_executable_hotgym} examples/Hotgym.cpp)
+add_executable(${src_executable_hotgym} examples/hotgym/Hotgym.cpp examples/hotgym/HelloSPTP.hpp)
 if(MSVC)
   # for Windows, link with the static library
   target_link_libraries(${src_executable_hotgym} 
-        ${core_library}
+    ${core_library}
 		${COMMON_OS_LIBS}
 		)
 else()
   # for Linux,OSx, link with the shared library
   target_link_libraries(${src_executable_hotgym} 
-        ${src_lib_shared}
+    ${src_lib_shared}
```

Comment:
Tabs show askew in diff.  Can you set your editor to replace tabs with spaces, then set tabs at 4 for CMake?

Reply:
vim started to misbehave, i'll try to fix that. sorry,